### PR TITLE
Adjust logcheck

### PR DESCRIPTION
**Pull request by 2Kmm**
Some little tweaks I personally use. Please see, if you like them. :)

### CODE REVIEW

**Comment by juippis** [comment]
I don't see those requests, they didn't get through (at least in this PR)

File: app-admin/logcheck/logcheck-1.4.3-r2.ebuild [hunk 1]
```diff
@@ -68,6 +68,8 @@ src_install() {
 	if use cron; then
 		exeinto /etc/cron.hourly
 		newexe "${FILESDIR}"/${PN}.cron ${PN}
+		insinto /etc/cron.d
```
[the remark this answers is not in the document]
Comment (Flowdalic) [reply app-admin/logcheck/logcheck-1.4.3-r2.ebuild:71]: It is 🙃

File: app-admin/logcheck/files/logcheck.cron.d [hunk 2]
```diff
@@ -0,0 +1 @@
+@reboot root sleep 60; /etc/cron.hourly/logcheck -R
```
[the remark this answers is not in the document]
Comment (Flowdalic) [reply app-admin/logcheck/files/logcheck.cron.d:1]: Is it even necessary to explicitly run this as 'root'? I had assumed that this is the default behavior, maybe simply

```suggestion
@reboot sleep 60; /etc/cron.hourly/logcheck -R
```

is sufficient.

[the remark this answers is not in the document]
Comment (2Kmm) [reply app-admin/logcheck/files/logcheck.cron.d:1]: I use cronie currently. From the manpage:
```
Jobs in /etc/cron.d/
       The jobs in cron.d and /etc/crontab are system jobs, which are used usually for more than one user,  thus,  additionally  the  username  is  needed.
```
This does not read, like the username is optional.

As [PFL](https://www.portagefilelist.de/?fs=%2Fetc%2Fcron.d%2F*) hints, there are several packages, that install to /etc/cron.d. I looked at some and all provide the user field.

See also: [Gentoo Forums - Weird cron email](https://forums.gentoo.org/viewtopic-t-1094390-start-0.html)

As I understand it, this is an incompatibility between dcron and all other cron implementations, where there is no solution that works for all. How do we want to handle this?

**Review by juippis — COMMENTED** [review]
Although could you please revbump and reset keywords? Make a new copy and don't touch current stable.
**ekeyword ~all logcheck-1.4.3-r3.ebuild**

**Comment by MocioF** [comment]
I have included Manuel Mommertz enhancements in https://github.com/gentoo/gentoo/pull/35681

**Comment by 2Kmm** [comment]
> Although could you please revbump and reset keywords? Make a new copy and don't touch current stable. **ekeyword ~all logcheck-1.4.3-r3.ebuild**

Oh, I was to slow. Sorry for that and thanks to @MocioF for bringing the changes to the tree. :+1:

**Pull request closed without merging**